### DISCUSSION TITLE
Issue with 'Get-OVMigratableServer' cmd

### DISCUSSION THREAD

**Discussion by FredSOS** [HewlettPackard/POSH-HPOneView#674] [Q&A]
Hi,

I've got the following issue when running 'Get-OVMigratableServer' cmd:

$MigratablesServers = Get-OVMigratableServer -Hostname $mySourceOVip -Credentials $myCredentials

**Get-OVMigratableServer: The source appliance address resolved to this appliance.  Provide a valid source appliance IP address or host name**

$mySourceOVip contains reachable OVhostname:443 and I can login through the WEB ui without a problem.

An idea?

Fred

**Comment by VincentBerger** [comment]
Server migration assumes 2 distinct OneView appliances: a source and a target. This cmdlet assumes you are connected to the **target** appliance through Connect-OVMgmt and you provide the **source** appliance IP in a parameter to the cmdlet. The error message appears to mean you used the same appliance for both.

**Comment by FredSOS** [comment]
Thanks a lot Vincent!

Question: is there a way to filter out the servers I want to migrate ?

**Comment by VincentBerger** [comment]
Standard Powershell pipelining should help. Something along the lines of:
Get-OVMigratableServer -... | Where-Object Name -match "your_criteria" | Invoke-OVMigrateServer -...
